rosbridge_library/src/rosbridge_library/internal/message_conversion.py

```diff
@@ -316,6 +331,10 @@ def _to_object_inst(msg, rostype, roottype, inst, stack):
     if rostype in ros_header_types:
         inst.stamp = ROSClock().now().to_msg()
 
+    # Do we have a dedicated decoder for the object?
+    if rostype in shortcut_object_decoders:
+        return shortcut_object_decoders[rostype](msg, inst)
+
```

Comment:
@srubs out of curiousity, this is the part of the code that the reviewer upstream had seemed to object to.

If the upstream PR is stagnant much longer, I may re-submit a new PR that omits this shortcut step so it can get the conversation moving forward again.

Reply:
Yeah. I'm all for that. The more of the official release that we make use of the better.